content/python/concepts/classes/classes.md

```diff
@@ -71,42 +78,36 @@ blue_home = Home()
 blue_home.paint_wall("blue")
```

Comment:
It might be good to provide something for the method body above (setting an attribute perhaps.)

Reply:
I ended up ditching the `Home` class/`.paint_wall()` example and replaced with a `feed_bird()` method inside the `Bird` class, instead.